scoringrules/_interval.py

```diff
@@ -0,0 +1,170 @@
+import typing as tp
+from typing import Optional, Union
+
+from scoringrules.backend import backends
+from scoringrules.core import interval
+
+if tp.TYPE_CHECKING:
+    from scoringrules.core.typing import Array, ArrayLike, Backend
+
+
+def interval_score(
+    observations: "ArrayLike",
+    lower: "Array",
+    upper: "Array",
+    alpha: Union[float, "Array"],
+    /,
+    axis: int = -1,
+    *,
+    backend: "Backend" = None,
+) -> "Array":
+    r"""Compute the Interval Score or Winkler Score [(Gneiting & Raftery, 2012)](https://www.tandfonline.com/doi/abs/10.1198/016214506000001437) for 1 - $\alpha$ prediction intervals PI = [lower, upper].
+
+    The interval score is defined as
+
+    $\text{IS} = \begin{cases}
+    (u - l) + 2\frac{2}{\alpha}(l - y)  & \text{for } y < l \\
```

Comment:
There is an extra "2" on line 26, same thing in other places

Reply:
Removed in the main docs and the interval_score() :+1: